src/asset.rs

```diff
@@ -568,6 +571,14 @@ impl BinRead for Asset {
             SPRING_BONE_MONO_BEHAVIOR_HASH => {
                 MonoBehavior::<SpringBone>::read_options(reader, endian, ()).map(Self::SpringBone)
             }
+            ANIMATION_CLIP_HASH => {
+                AnimationClip::read_options(reader, endian, ()).map(Self::AnimationClip)
+             }
+             _ => Ok(Self::Unparsed(Unparsed {
+                 type_hash,
+                 path_id: pptr,
+                 blob: binrw::until_eof(reader, endian, ())?,
+             })),
```

Comment:
Can you get rid of this? Looks like we ended up with 2 default clauses somehow.

Reply:
Done, thanks.